agent/src/beerocks/monitor/monitor_db.cpp

```diff
@@ -1,6 +1,6 @@
 /* SPDX-License-Identifier: BSD-2-Clause-Patent
  *
- * Copyright (c) 2016-2019 Intel Corporation
+ * SPDX-FileCopyrightText: 2016-2019 the prplMesh contributors (see AUTHORS.md)
```

Comment:
What about years? You have modified file in 2020, but did not fix the years, it is still 2016-2019

Reply:
Indeed, I didn't check that for every file...